metricbeat/metricbeat.reference.yml

```diff
@@ -154,6 +154,14 @@ metricbeat.modules:
   hosts: ["localhost:5000"]
   enabled: true
 
+#------------------------------- consul Module -------------------------------
+- module: consul
+  metricsets: ["agent"]
+  enabled: false
```

Comment:
I think in the reference file we have enabled: true by default.

Reply:
Set `enabled = true`